db/seeds.rb

```diff
@@ -234,101 +192,23 @@ def generate_checkout
   end
 end
 
-def generate_blackout
-  Blackout.create! do |blk|
-    blk.start_date = time_rand(Time.zone.now + 1.year)
-    # rubocop:disable Date
-    blk.end_date = time_rand(blk.start_date.to_time,
```

Comment:
Why were we using `to_time` before? I'm assuming you've noticed no issues since removing it?